rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/RSpace.scala

```diff
@@ -468,6 +469,32 @@ object RSpace {
 
   }
 
+  def createWithReplay[F[_], C, P, A, R, K](dataDir: Path, mapSize: Long)(
```

Comment:
this looks unused, but I assume the usage will come later? Maybe it'd make sense to commit that with the usage? Or a test at least?

Reply:
```
F[(RhoContext[F], RhoISpace[F], RhoReplayISpace[F])] =
      for {
        withReplay <- NextRSpace.createWithReplay[
```?

Reply:
lol :D